doc/how_to_build_win.md

```diff
@@ -34,9 +34,10 @@ They will have to be installed seperately as follows.
 - Install Qt 5.9 (64-bit version) by [Qt Online Installer for Windows](http://download.qt.io/official_releases/online_installers/qt-unified-windows-x86-online.exe).
 
 ### boost
-- Boost 1.55.0 or later is required (tested up to 1.60.0).
+- Boost 1.55.0 or later is required (tested up to 1.61.0).
 - http://www.boost.org/users/history/version_1_61_0.html
 - Download boost_1_61_0.zip from the above link.  Extract all contents to the '$opentoonz/thirdparty/boost' directory.
+- Rename the folder from "boost_1_61_0" to "boost_1_55_0".
```

Comment:
This shouldn't be necessary.

Reply:
I didn't know that, sorry! So CMake will take care of it? Are newer versions working?

Reply:
Sorry for the double reply. If you don't rename, does an error appear in CMake as with Qt? Because if so, I will note it in the instructions.